crates/sui-move/src/main.rs

```diff
@@ -26,10 +26,18 @@ struct Args {
     #[clap(long = "path", short = 'p', global = true)]
     pub package_path: Option<PathBuf>,
     /// If true, run the Move bytecode verifier on the bytecode from a successful build
-    #[clap(long = "path", short = 'p', global = true)]
+    #[clap(
+        long,
+        default_value_if("package_path", builder::ArgPredicate::IsPresent, Some("true")),
```

Comment:
```suggestion
        
```
I don't think this condition is really needed. The way I understand this flag is that it should be set to `global` true regardless of `package_path` option.